pkg/sysinfo/sysinfo.go

```diff
@@ -35,6 +36,12 @@ func New(quiet bool) *SysInfo {
 		if !sysInfo.SwapLimit && !quiet {
 			log.Printf("WARNING: Your kernel does not support cgroup swap limit.")
 		}
+
+		_, err = ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.oom_control"))
+		sysInfo.OomKillDisable = err == nil
+		if !sysInfo.OomKillDisable && !quiet {
+			log.Printf("WARNING: Your kernel does not support oom control.")
+		}
```

Comment:
memory.oom_control is with memory cgroup, I think it is not needed to check this file separately.